source/plugins/music/index.mjs

```diff
@@ -502,7 +502,7 @@ export default async function({login, imports, data, q, account}, {enabled = fal
   }
 }
 
-//get all objects that have the given key name with recursivity
+//get all objects that have the given key name with recursively
```

Comment:
?

Reply:
This comment was probably written by someone else since it doesn't start with an uppercase, but I guess they just meant they were doing it recursively. 

Removing `with` seems to make sense